docs/api.md

```diff
@@ -0,0 +1,233 @@
+# electron-packager API
+
+Short example:
+
+```javascript
+var packager = require('electron-packager')
+packager(options, function done_callback (err, appPath) { /* … */ })
```

Comment:
The callback receives an array, since 5.0.0.  (I guess I missed updating something in the readme back when I did #88?)